pocs/mount/bisque.py

```diff
@@ -301,14 +302,14 @@ def write(self, value):
         return self.theskyx.write(value)
 
     def read(self, timeout=5):
```

Comment:
Why not just change the default to 120 seconds here?

We could still feasibly allow for timeouts for each method via the keywords you have added or just pass the `kwargs` through for each command method.

Reply:
Basically because a 5s timeout is all we really need for the other (non-movement related) SkyX operations.

Reply:
> I don't see the "removes the redundant timeout functionality in `pocs.mount.bisque.read`" part of this.
> 
> I'm approving as well but see my note about the default timeout. Could probably just change the default timeout and not bother with the rest of the PR or do as note suggests.

@wtgee Did you mean to approve?